### DISCUSSION TITLE
Getting Trips?

### DISCUSSION THREAD

**Discussion by GeraldPape** [flobz/psa_car_controller#623] [General]
No matter what I tried I do not get any trips. Only via bluetooth on my mobile. So I guess I missed something, or did something wrong.
Was anyone able to get trips into the tool and could share his wisdom?

Thank you!

Jerry

**Comment by GeraldPape** [comment]
No One? Is this feature even working?

**Reply by sod75** [reply]
worksforme
did you enable the privacy option to send data ( I had to in my peugeot) and I also configured my home wifi  fro when it's in my driveway

**Reply by GeraldPape** [reply]
Yes, privacy settings would allow. 
Also have the home wifi configured, the car can use it. 
But still no trips
